modular_zubbers/modules/primitive_cooking/code/butterchurn.dm

```diff
@@ -0,0 +1,102 @@
+#define CHURN_CREAM_RATIO 3 //3:1 milk:cream
+#define CHURN_BUTTER_RATIO 10 // 10:1 cream:butter. Slightly worse than using technology.
+#define CHURN_STAMINA_MINIMUM 51 //Can use it twice before you really gotta rest.
+#define CHURN_STAMINA_USE 50 //Half that of the millstone, since making butter needs you to use it twice.
+
+/obj/structure/butterchurn
+	name = "butter churn"
+	desc = "A small wooden barrel with a stick that thickens milk into a rich cream, or even butter. Just add an absolutely inordinate amount of effort."
+	icon = 'modular_zubbers/icons/obj/structures/butterchurn.dmi'
+	icon_state = "butterchurn"
+	density = TRUE
+	anchored = TRUE
+	max_integrity = 120
+	pass_flags = PASSTABLE
+	resistance_flags = FLAMMABLE
+	custom_materials = list(
+		/datum/material/wood = SHEET_MATERIAL_AMOUNT  * 8,
+	)
+	drag_slowdown = 2
+	var/busy = FALSE
+
+/obj/structure/butterchurn/Initialize(mapload)
+	. = ..()
+	create_reagents(100, OPENCONTAINER)
+
+/obj/structure/butterchurn/examine(mob/user)
+	. = ..()
+	. += span_notice("You can [anchored ? "un" : ""]secure [src] with <b>CTRL-Shift-Click</b>.")
+	. += span_notice("With a <b>prying tool</b> of some sort, you could take [src] apart.")
```

Comment:
Would appreciate if these interactions were added as screentips too

Reply:
Added tooltip context in latest commit!